src/main/xml/options/soot_options.xml

```diff
@@ -542,6 +545,23 @@
                 </p>
             </long_desc>
         </listopt>
+        <boolopt>
+            <name>Derive Java version from input</name>
+            <alias>derive-java-version</alias>
```

Comment:
@pavanupb This is the code I actually added manually. From this xml document the options.html and some Java classes relating to the option will be automatically created by Maven. if you merged my changes from options.xml into your branch, you should thus get all the Java code related to `derive-java-version`